src/main.rs

```diff
@@ -27,12 +27,12 @@ const USAGE: &'static str = "
 HEMTT, a simple to use build manager for Arma 3 mods using the CBA project structure
 
 Usage:
-    hemtt init
-    hemtt create
-    hemtt addon <name>
-    hemtt build [--release] [--force] [--nowarn] [--opts [<optionals>]]
-    hemtt clean [--force]
-    hemtt run <utility>
+    hemtt init [-v]
+    hemtt create [-v]
+    hemtt addon [-v] <name>
+    hemtt build [-v] [--release] [--force] [--nowarn] [--opts [<optionals>]]
+    hemtt clean [-v] [--force]
+    hemtt run [-v] <utility>
```

Comment:
None I could find. You can do [options], but then all will have all flags.

Reply:
Yeah I couldn't find anything either. I tried creating a new section `flags` and using `[flags]` but that didn't work. I'm going to do some more digging today, but this may be the only option unfortunately.